js/main.js

```diff
@@ -82,7 +82,9 @@ function addEntry(entry) {
 
 function contentLoadHandler(event) {
   for (var i = 0; i < data.entries.length; i++) {
-    ul.appendChild(addEntry(data.entries[i]));
+    if (data.entries[i] !== null) {
```

Comment:
Why is this necessary? Do you have `null` in your array of entries?

Reply:
I did somehow but I removed it, I forgot to take this part out after.